src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/AlternativesToRecallTried.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.recommendation
+
+data class AlternativesToRecallTried(
+  val selected: List<SelectedAlternative>?,
+  val allOptions: List<RecallAlternative>?
+)
+
+data class SelectedAlternative(
+  val value: String?,
+  val details: String?
+)
+
+data class RecallAlternative(
```

Comment:
I think @jackmaloney created a shared type for this already

Reply:
Done ...twice!!